<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lMapDir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current map-path:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lMapDirStr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;none selected&gt;

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetMapDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose map directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load/Refresh maps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreateNewDB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create new database</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetFilters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Filters</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum max players:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max players:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMinimumRating">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>30</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMinimumMaxPlayers">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>40</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbMaxPlayers">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>40</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File-name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingame-name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tags:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFilterFileName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>109</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFilterIngameName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>89</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leFilterTags">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>139</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenInExplorer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open map in explorer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoveMap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Move map to current map-
path and move all other maps
 in map-path to a backup
 folder</string>
   </property>
  </widget>
  <widget class="QTableView" name="view">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>0</y>
     <width>231</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
